src/actions/view-party-settings.ts

```diff
@@ -165,15 +153,8 @@ export async function insertPlaylist(
             const resp = await fetchWithAccessToken(url);
             const { items, next }: SpotifyApi.PlaylistTrackResponse = await resp.json();
             const trackItems = items
-                .filter(
-                    it =>
-                        it &&
-                        !it.is_local &&
-                        it.track &&
-                        it.track.id &&
-                        it.track.is_playable !== false,
-                )
-                .map(it => it.track);
+                .filter((it) => it && !it.is_local && it.track.id && it.track.is_playable !== false)
```

Comment:
This is missing the check for `it.track`, is this intentional?

Reply:
The Spotify types specify that `track` is never undefined, so I removed the check for it. In case it might be undefined, the check for `it.track.id` would resolve to false.